### ISSUE TITLE
Jumbo cactpot claims that there are rewards

### ISSUE THREAD

**Issue by gurkburk76** [MidoriKami/DailyDuty#30]
Hello, so i was sure to isntall this plugin BEFORE i got my 3 jumbo  tickets but i still after collecting my 3 consolation prizes still this in chat:
"[DailyDuty] [Jumbo Cactpot] 1 Rewards Available"
Waht can i do to help you debug the issue?

**Comment by MidoriKami** [comment]
Send config, and describe the sequence of events in as much detail as you can when you went to collect your rewards

**Comment by gurkburk76** [comment]
I'm assuming the config is from here:
........AppData\Roaming\XIVLauncher\pluginConfigs\DailyDuty.json

and exactly what i did is from what i can remember is to speak to the "cactpot cashier", perhaps multiple times.
It is within the realm of possibilities that i spoke to the "jumbo cactpot broker" before as well, first time for me being there and all.

Tried to ul the json file, funny how the drag & drop function dosen't seem to like that kind of file.
[DailyDuty.json.log](https://github.com/MidoriKami/DailyDuty/files/8163391/DailyDuty.json.log)

**Comment by MidoriKami** [comment]
Did you try to buy a ticket and then go nevermind and close the window when it asked you to pick 4 digits?

**Comment by gurkburk76** [comment]
I don't think so.

**Comment by MidoriKami** [comment]
If the ticket purchase window is opened, it considers that to be the requirement for buying a ticket. I found a new method for check for completion of an action from implementing Doman Enclave, I plan to add those improvements to the other modules, so this won't be a problem in the future.

For now you can manually correct the configuration file while the game is **closed**
and remove the following data:
```
          {
            "$type": "DailyDuty.Data.ModuleData.JumboCactpot.TicketData, DailyDuty",
            "CollectedDate": "2022-02-25T07:59:11.0852019Z",
            "DrawingAvailableTime": "2022-02-26T19:00:00Z",
            "ExpirationDate": "2022-03-05T19:00:00Z"
          },
```

The ticket will automatically be deleted after the expiration date on the 5th.

I'll keep this issue open until I have implemented the system changes.

**Comment by MidoriKami** [comment]
Fixed in 7a4a313fa6924be033cd76ec00b1248a29cfe9cf
